Algorithm.CSharp/OpeningBreakoutAlgorithm.cs

```diff
@@ -288,7 +288,7 @@ private void ScanForEntrance()
 
             var allowedDollarLoss = MaximumPorfolioRiskPercentPerPosition * Portfolio.TotalPortfolioValue;
 
-            var shares = (int) (allowedDollarLoss/expectedCaptureRange);
+            var shares = allowedDollarLoss/expectedCaptureRange;
```

Comment:
Please leave this as int cast; equities (via the brokerage model "can place order") should throw/reject order on fractional trade requests.